src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "shutdown (Android SDK API)"
+type: apiDoc
+shortDescription: Shut down the agent during runtime
+tags:
+  - Mobile monitoring
+  - New Relic Mobile Android
+  - Android SDK API
+metaDescription: 'New Relic API for Android mobile app monitoring: shut down agent.'
+redirects:
+  - /docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/shut-down
+---
+
+## Syntax
+
+```
+NewRelic.shutdown()
+```
+
+Shut down the agent within the current application lifecycle during runtime
+
+## Requirements
+
+Agent version 6.10.0 or higher.
+
+## Description
+
+This API provides the ability to shutdown the agent within the current application lifecycle during runtime. Once the agent has shutdown, it cannot be restarted within the current application lifecycle.
```

Comment:
```suggestion
This API provides the ability to shut down the agent within the current application lifecycle during runtime. Once the agent has shut down, it cannot be restarted within the current application lifecycle.
```

Reply:
@ywang-nr What's a scenario where the user would want to use this API?

Reply:
This feature is meant to give the developer options to disable our agent, most likely related to their end users' privacy issue